app/code/community/ShipStream/Sync/Model/Cron.php

```diff
@@ -90,7 +101,34 @@ protected function _getTargetInventory(array $skus)
             ->where('si.stock_id = ?', Mage_CatalogInventory_Model_Stock::DEFAULT_STOCK_ID)
             ->where('p.sku IN (?)', $skus);
 
-        // TODO - subtract processing that is not "submitted"
+        return $db->fetchAssoc($select);
+    }
+
+    /**
+     * Retrieve Magento order items qty that are in processing state and not submitted to shipstream
+     * @param array $sku
+     * @return array
+     */
+    protected function _getProcessingOrderItemsQty(array $skus): array
+    {
+
+        $orderStates = [
+            Mage_Sales_Model_Order::STATE_COMPLETE,
+            Mage_Sales_Model_Order::STATE_CLOSED,
+            Mage_Sales_Model_Order::STATE_CANCELED
+        ];
+        $resource = Mage::getSingleton('core/resource');
+        $db = $resource->getConnection('core_write');
+        $columns = ['sku' => 'soi.sku', 'qty' => 'sum(soi.qty_ordered)'];
+        $select = $db->select()->forUpdate(TRUE)
+            ->from(['soi' => $resource->getTableName('sales/order_item')], $columns)
+            ->join(['so' => $resource->getTableName('sales/order')], 'so.entity_id = soi.order_id', [])
+            ->where('so.state NOT IN (?)',$orderStates)
+            ->where('so.status != (?)',"submitted")
+            ->where('so.hold_before_status != (?)',"submitted")
+            ->where('soi.sku IN (?)', $skus)
+            ->where('soi.product_type = (?)', 'simple')
```

Comment:
I've never seen the right hand side wrapped in parentheses for = or !=, looks odd..